posts/2019/farmOS-1.0/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: farmOS 1.0
+date: 2019-03-20
+author: Michael Stenta
+slug: 2019/farmOS-1.0
+---
+
+# farmOS 2.0
```

Comment:
```suggestion
# farmOS 1.0
```
![image](https://github.com/farmOS/farmOS-community-blog/assets/52507690/d6dcf3ff-b884-45bb-986f-82a76f0f0d9b)

Reply:
Good catch @wotnak! My fingers only know how to type "2.0" now. :-)

I'll push a fixup commit...